client/src/index.js

```diff
@@ -12,14 +12,65 @@ import "./index.scss";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import rootReducer from "./reducers";
-
+import zIndex from "@material-ui/core/styles/zIndex";
```

Comment:
even though we currently don't have a lint rule for this in the frontend, for the imports I think it's best to group them, first all the external dependencies and then the internal own to our own code... makes it a bit more clear, I would even prefer if they are done alphabetically as well so it's easy to find something.